builtin/logical/database/dbplugin/client.go

```diff
@@ -37,9 +37,7 @@ func NewPluginClient(ctx context.Context, sys pluginutil.RunnerUtil, pluginRunne
 	pluginSets := map[int]plugin.PluginSet{
 		// Version 3 supports both protocols
```

Comment:
Should we update this comment? Version 3 still exists, but it won't really be supported after this PR.

Reply:
Yep good call